script/Ciel/Day 5/05_01_CIEL05_1D.txt

```diff
@@ -123,57 +147,65 @@ A %{i}normal%{/i} human would go insane! There's no way you could be used to see
 [sha:dec1835ff5e14c27953e6be1d57fd7e8151acb0f]{
 -- Page 161, Offset 11308.
 -- 息が詰まる。
-I have trouble breathing.
+My breath catches in my throat.
 }
 [sha:f0ae687acf1a3f7af8909ce50cd4f8289cd5971a]{
 -- Page 161, Offset 11309.
 -- <女|ノエル>の泣き声は、<感情的|ヒステリック>でありながら、的確に俺の矛盾を指摘していた。
-Noel-sensei's crying voice, hysterical as it sounds, rightly points out the contradiction.
+Noel-sensei may be screaming hysterically,
+ but she's still incisively called out my abnormal behavior.
 }
 [sha:08c04cc9bc248389e164da91cbbcabc58dc1beac]{
 -- Page 162, Offset 11310.
 -- あり得ない空間。
-An impossible space.
+This is an unimaginable space.
 }
 [sha:695fe34d82a2bca1f4bc25c239843b03dca5ecac]{
 -- Page 162, Offset 11311.
 -- 蟻の巣のように張り巡らされた地下墓地。動く死体たち。
-An underground graveyard stretching out like an ant nest. Walking dead bodies.
+A sprawling underground graveyard that resembles some kind of ant nest.
```

Comment:
i believe they're called ant colonies

Reply:
Funnily enough that's what I thought too but I actually googled it to be sure [(wiki)](https://en.wikipedia.org/wiki/Ant_colony#Names)
> The term "ant colony" refers to a population of workers, reproductive individuals, and brood that live together, cooperate, and treat one another non-aggressively. [...] "Ant nests" are the physical spaces in which the ants live. These can be underground, in trees, under rocks, or even inside a single acorn.[[6]]